docs/02-app/02-api-reference/01-components/link.mdx

```diff
@@ -194,8 +213,38 @@ export default function Page() {
 }
 ```
 
+</AppOnly>
+
 <PagesOnly>
 
+**Defaults to `true`.** When `true`, `next/link` will prefetch the page (denoted by the `href`) in the background. This improves the performance of client-side navigations. Any `<Link />` in the viewport (initially or through scroll) will be prefetched and loaded in the background.
+
+Prefetch can be disabled by passing `prefetch={false}`. Prefetching is only enabled in production.
```

Comment:
The `Prefetching is only enabled in production.` feels like it belongs with the `true` statement instead of the `false.